tracing-core/src/subscriber.rs

```diff
@@ -314,6 +314,24 @@ pub trait Subscriber: 'static {
         let _ = id;
     }
 
+    /// Returns a type representing the span that represents this subscriber's
+    /// view of the "current" span.
+    ///
+    /// If subscribers track a current span, they should override this function
```

Comment:
I'm having a bit of trouble phrasing this paragraph (lines 320–323). Can you try to rephrase?